_posts/rfc/2016-04-01-PI-filter.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: API Privacy Index filtering mechanism
+category: rfc
+status: proposal
+---
+
+This RFC describe how Privacy Index filtering is used between CaliOpen ReST API
+and a Client.
+
+## Resources
+
+Resources using Privacy Index filtering are **Thread**, **Message** and **Contact**.
+
+## Principle
+
+An HTTP Header **X-Caliopen-PI** is used to give PI range wanted by an API Client.
+Format for this header is an integer range between 0 and 100, inclusives values.
+Values are separated using semi colon ';' character.
+
+### Examples:
+
+X-Caliopen-PI: 0;100 -> to retrieve all resources
```

Comment:
what if resource has an undefined PI ?

* always omitted ?
* given if min value is 0 ?